### PR TITLE
support function name with underline and number

### PR DESCRIPTION
**Pull request by hyacker**
some function name may have _ or number that could be support

### CODE REVIEW

File: moz_sql_parser/sql_parser.py [hunk 1]
```diff
@@ -266,7 +266,7 @@ def to_string(instring, tokensStart, retTokens):
     (Literal("-")("op").setDebugActions(*debug) + expr("params")).addParseAction(to_json_call) |
     sqlString.setName("string").setDebugActions(*debug) |
     (
-        Word(alphas)("op").setName("function name").setDebugActions(*debug) +
+        Word(alphas + "_" + alphanums)("op").setName("function name").setDebugActions(*debug) +
```
Comment (klahnakoski) [inline moz_sql_parser/sql_parser.py]: Does 

    ident("op").setName("function name").setDebugActions(*debug) +

work instead?

**Comment by klahnakoski** [comment]
Please make a test for the particular function name that interests you

File: moz_sql_parser/sql_parser.py (hunk 1 above)
Comment (hyacker) [reply moz_sql_parser/sql_parser.py]: Yes, it works, and that will be more common.you do great job!

**Comment by klahnakoski** [comment]
It has been a while, and this is incomplete

**Pull request closed without merging**